lib/types/zpool.nix

```diff
@@ -1,6 +1,6 @@
 { config, options, lib, diskoLib, rootMountPoint, ... }:
 let
-  # TODO: Consider expanding to handle `disk` `file` and `draid` mode options.
+  # TODO: Consider expanding to handle `file` and `draid` mode options.
```

Comment:
It doesn't, or at least not as a keyword like `mirror`/`raidz`/etc. The `disk` section of the man page simply refers to passing in the path of a disk (e.g. `/dev/sda`), which is already supported.

This also applies to `file`, but I left that one since I'm not sure passing in a file is possible with the current setup.